src/main/java/VASSAL/configure/ConfigureTree.java

```diff
@@ -284,6 +321,205 @@ protected JPopupMenu buildPopupMenu(final Configurable target) {
     addAction(popup, buildImportAction(target));
     return popup;
   }
+  
+  
+  // Enumerates our configure tree in preparation for searching it
```

Comment:
Could make this a proper Javadoc, especially on the public methods down below, the syntax for these comments to be detected as Javadoc is `/** */` e.g.:
```
/**
 * A nice comment describing the method
 */
public void aMethod() {
```